openapi.yaml

```diff
@@ -294,6 +294,52 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/InvalidParametersError"
+    post:
+      tags:
+        - Alimentaire
+      summary: Calcul des impacts environnementaux d'une recette alimentaire
+      requestBody:
+        description: Requête modélisant les éléments de la recette à évaluer
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/FoodQuery"
+            examples:
+              CarrotCake:
+                summary: "Un gâteau de carottes"
+                value:
+                  ingredients:
+                  - id: egg
+                    mass: 0.12
+                  - id: wheat
+                    mass: 0.14
+                  - id: milk
+                    mass: 0.06
+                  - id: carrot
+                    mass: 0.225
```

Comment:
Could we have some example ingredients with more info, like country of origin, by plane, variant, bonuses...?